### ISSUE TITLE
Accessing ImpalaToGo with Tachyon

### ISSUE THREAD

**Issue by mrakshay** [ImpalaToGo/ImpalaToGo#85]
Hello , I do see that using ImpalaToGo we cannot access HDFS as ImpalaToGo has been separated from Hadoop. ImpalaToGo has been designed to work with Tachyon. As per the steps mentioned for [running ImpalaToGo over tachyon](https://github.com/ImpalaToGo/ImpalaToGo/wiki/To-run-ImpalaToGo-over-Tachyon) , I am not able to access tachyon file system. I see that if we manually install Tachyon, then there is lot of difference between the Tachyon which gets installed with Impala2Go & the Tachyon which gets installed manually..  How should I access & configure Tachyon for Impala2Go  ??

**Comment by navado** [comment]
ImpalaToGo designed to work without Hadoop caching remote HDFS on local resources.
Have you tried to start our image on AWS and select Tachion as caching layer as described in  [Quick Start](https://github.com/ImpalaToGo/ImpalaToGo/wiki/Quick-Start) guide?

**Comment by mrakshay** [comment]
Yes I selected the Tachyon as caching layer as described for CloudFormation template and I followed the same steps as mentioned in  [Quick Start](https://github.com/ImpalaToGo/ImpalaToGo/wiki/How-to-try)....but no success

**Comment by elenaviter** [comment]
Please provide the logs collected from your experiment
